.eslintrc.json

```diff
@@ -11,7 +11,10 @@
     "RequireJS": true
   },
   "parserOptions": {
-      "ecmaVersion": 5
+      "ecmaVersion": 5,
+      "ecmaFeatures": {
+        "globalReturn": false
```

Comment:
It appears to be a bug in ESLint that this is necessary - their docs say these should all be false by default, but things fail if this is not explicitly set.